### PR TITLE
add DMA configuration for `gr716-mimas`

### PR DESCRIPTION
**Pull request by lukileczo**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## How Has This Been Tested?

- [ ] Already covered by automatic testing.
- [ ] New test added: (add PR link here).
- [ ] Tested by hand on: (list targets here).

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing linter checks and tests passed.
- [x] My changes generate no new compilation warnings for any of the targets.

## Special treatment

- [ ] This PR needs additional PRs to work (list the PRs, preferably in merge-order).
- [ ] I will merge this PR by myself when appropriate.

### CODE REVIEW

File: _projects/sparcv8leon3-gr712rc-board/board_config.h [hunk 1]
```diff
@@ -59,6 +59,13 @@
 #define UART4_ACTIVE 0
 #define UART5_ACTIVE 0
 
+#define UART0_DMA 0
```
Comment (maska989) [inline _projects/sparcv8leon3-gr712rc-board/board_config.h:62]: To run DMA on sparcv8leon3-gr716-mimas, we must enable it on UART0. After that everything works perfectly.

**Review by maska989 — COMMENTED** [review]
Leaving small suggestion.

File: _projects/sparcv8leon3-gr712rc-board/board_config.h (hunk 1 above)
Comment (lukileczo) [reply _projects/sparcv8leon3-gr712rc-board/board_config.h:62]: You've marked the wrong file :) (for `gr712rc`). DMA is enabled on the console UART1 on `gr716-mimas` (https://github.com/phoenix-rtos/phoenix-rtos-project/blob/34b7b7dd130a297d2fcc7222d2d027163159dca7/_projects/sparcv8leon3-gr716-mimas/board_config.h#L63), should it be on UART0 instead?

Comment (maska989) [reply _projects/sparcv8leon3-gr712rc-board/board_config.h:62]: You are right :+1: Yes, it should be on UART0. My bad.

Comment (lukileczo) [reply _projects/sparcv8leon3-gr712rc-board/board_config.h:62]: Done :+1:

**Review by agkaminski — APPROVED** [review]

**Pull request merged**